backend/server.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express'); 
+const app = express();
+const port = 8080;
+const fs = require('fs');
+app.use(express.json()); 
+
+app.get('/', (request, response) => { 
+    response.send ('my roots?');
+});
+
+app.post('/sign-up', (request,response) => { // Sign up route, get the new user data 
+    const email = request.body.email; 
+    const pass = request.body.password;
+    const confPass = request.body.confPass;
```

Comment:
this is extra credit but consider doing this using [object destructuring](https://dmitripavlutin.com/javascript-object-destructuring/).

```
const { email, password, confPass } = request.body;
```